minecraft/dial.go

```diff
@@ -290,12 +294,14 @@ func listenConn(conn *Conn, logger *log.Logger, l, c chan struct{}) {
 			if !errors.Is(err, net.ErrClosed) {
 				logger.Printf("dialer conn: %v\n", err)
 			}
+			e <- err
```

Comment:
If this (or the next `e <- err`) is reached after the connection is created (DialContext is done running), it will block the goroutine forever because there's nothing to take the error out of the channel.

Reply:
i think i (cursor + claude sonnet) fixed it in the latest commit